test/unit/src/common/reducers/entities/fixtures.js

```diff
@@ -0,0 +1,64 @@
+export const initialState = {
+  repos: {},
+  snaps: {},
+  owners: {}
+};
+
+export const payload = {
+  entities: {
+    repos: {
+      1001: {
+        owner: 1,
+        fullName: 'foo'
+      },
+      1002: {
+        owner: 1,
+        fullName: 'bar'
+      },
+      1003: {
+        owner: 1,
+        fullName: 'baz'
+      }
+    },
+    owners: {
+      1: {
+        name: 'Canonical'
+      }
+    }
+  },
+  result: [ 1001,1002,1003 ]
+};
+
+export const finalState = {
+  ...initialState,
+  ...payload.entities
+};
+
+export const repoPayload = {
+  id: 1001
+};
+
+export const repoAddState = {
+  isSelected: true,
+  isFetching: true,
+  error: null
+};
+
+export const repoSuccessState = {
+  isSelected: false,
+  isFetching: false,
+  error: null
+};
+
+export const repoFailureState = {
+  isSelected: true,
```

Comment:
This should arguably be false, allowing the user to submit the remaining repos without further interaction.